app/deck/1.8.x/guides/best-practices.md

```diff
@@ -4,41 +4,39 @@ toc: false
 - Always use a pinned version of decK and Kong.
-  Achieving declarative configuration is
-  not easy because details matter a lot. Use a specific version of decK in
-  production. If you're going to start using a new version of decK or Kong,
+  Use a specific version of decK in
+  production to achieve declarative configuration. If you're going to opt for a new version of decK or Kong,
   please safely test the changes in a staging environment first.
-- decK does not manage encryption of sensitive information, meaning private
-  keys of your certificates and credentials of consumers will be stored in
-  plaintext in the state file. Please be careful in how and where you store
-  this file as it can have a huge impact on your security.
-  You should store these in an encrypted form and provide a plaintext version
-  of this on a need-by-need basis.
-- If you have a very large number of consumers in your database, do not export
-  or manage them using decK. Declarative configuration is for ... configuration,
-  it is not meant for end-user data, which can easily grow into hundreds of
-  thousands or millions.
+- decK does not manage encryption of sensitive information. The state file stores the private
+  keys of your certificates and credentials of consumers in
+  plaintext. Please be careful in how and where you store
+  this file to avoid any security breaches.
+  You should always store the sensitive information in an encrypted form and provide a plaintext version
+  of it on a need-only basis.
+- If you have many consumers in your database, do not export
+  or manage them using decK. Declarative configuration is only for configuration.
+  It is not meant for the end-user data which can easily grow into hundreds of thousands or millions.
 - Always run a `deck diff` command before running a `deck sync`
-  to ensure the change that is taking place.
+  to ensure that the change works properly.
 - Adopt a [CI-driven configuration](/deck/{{page.kong_version}}/guides/ci-driven-configuration) practice.
-- Always secure Kong's Admin API with some kind of authentication method.
-- Do not write the state file by hand, it will be very error-prone.
-  Instead, use Kong's Admin API to
-  configure Kong for the first time, then export the configuration. Any
+- Always secure Kong's Admin API with a reliable authentication method.
+- Do not write the state file by hand to avoid errors.
+  Always use Kong's Admin API to
+  configure Kong for the first time and then export the configuration. Any
   subsequent changes should be made by manually editing the file and pushing
-  the change via CI. If you're making a large change, make the change in Kong,
-  export the new file, and then diff the two state files to review the changes
+  the change via CI. If you're making a larger change, make the changes in Kong and
+  export the new file. Then you can differentiate the two state files to review the changes
```

Comment:
```suggestion
  export the new file. Then you can `diff` the two state files to review the changes
```
diff is fine to use here, it's the specific command name.

Reply:
Okay got it.